pkgs/applications/networking/shellhub-agent/fix-go-mod-deps.patch

```diff
@@ -0,0 +1,128 @@
+diff --git a/agent/go.mod b/agent/go.mod
+index c075083..b79726e 100644
+--- a/agent/go.mod
++++ b/agent/go.mod
+@@ -28,6 +28,7 @@ require (
+ 	github.com/pkg/errors v0.9.1
```

Comment:
is this appropriate to upstream? seems non-nixpkgs specific

Reply:
We should probably upstream this patch and fetch it.

Reply:
This will not be part of new release as this cannot be upstreamed anymore. We are finishing 0.7.0 release now.

Reply:
https://github.com/shellhub-io/shellhub/pull/908 ;-)

Reply:
If it's being addressed, then I think that's fine :)